source/dynamical_systems/include/dynamical_systems/Linear.hpp

```diff
@@ -107,9 +114,15 @@ class Linear : public DynamicalSystem<S> {
    * @brief Return a list of all the parameters of the dynamical system
    * @return the list of parameters
    */
-  std::list<std::shared_ptr<state_representation::ParameterInterface>> get_parameters() const;
+  std::list<std::shared_ptr<ParameterInterface>> get_parameters() const override;
 };
 
+template<>
+void Linear<JointState>::set_gain(double iso_gain) {
```

Comment:
if you put a specialized version in the header then it should be because you want to `inline` it. Otherwise it should go in the source file.

Reply:
I needed to put it here because it's used in the subsequent `set_attractor` for the `JointState`. But I forgot to `inline` it